src/components/Navbar/NavbarButtons/WalletManagementButtons.tsx

```diff
@@ -140,15 +136,28 @@ export const WalletManagementButtons: React.FC<
       onClick={handleWalletMenuClick}
     >
       {isSuccess && activeChain ? (
-        <WalletMgmtAvatarContainer>
+        <Badge
+          overlap="circular"
+          className="badge"
+          anchorOrigin={{ vertical: 'bottom', horizontal: 'right' }}
+          badgeContent={
+            <Avatar
+              size="small"
+              src={activeChain.logoURI || 'empty'}
+              alt={`${activeChain.name}chain-logo`}
+            >
+              {activeChain.name[0]}
+            </Avatar>
+          }
+        >
           <WalletMgmtWalletAvatar src={walletIcon} />
-          <WalletMgmtChainAvatar
-            src={activeChain.logoURI || 'empty'}
-            alt={`${activeChain.name}chain-logo`}
-          />
-        </WalletMgmtAvatarContainer>
+        </Badge>
       ) : null}
-      <Typography variant={'lifiBodyMediumStrong'} width={'auto'}>
+      <Typography
+        variant={'lifiBodyMediumStrong'}
+        width={'auto'}
+        sx={{ marginLeft: theme.spacing(1) }}
```

Comment:
We can use numbers here, they will be automatically converted to the theme.spacing values.

Reply:
I had no idea! Will give this a try